containers/doc/server-kubernetes/migration-job.yaml

```diff
@@ -39,6 +39,7 @@ spec:
                           /etc/salt \
                           /etc/tomcat \
                           /etc/cobbler \
+                          /etc/postfix \
```

Comment:
No longer needed now that we are moving the feature to `uyuniadm`. We can drop the whole file.

Reply:
File removed